src/main/java/nextstep/fp/Lambda.java

```diff
@@ -26,31 +26,7 @@ public void run() {
         }).start();
     }
 
-    public static int sumAll(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            total += number;
-        }
-        return total;
-    }
-
-    public static int sumAllEven(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            if (number % 2 == 0) {
-                total += number;
-            }
-        }
-        return total;
-    }
-
-    public static int sumAllOverThree(List<Integer> numbers) {
-        int total = 0;
-        for (int number : numbers) {
-            if (number > 3) {
-                total += number;
-            }
-        }
-        return total;
+    public static int sum(List<Integer> numbers, SumStrategy sumStrategy) {
+        return numbers.stream().filter(sumStrategy::isSuitable).reduce(0, (x, y) -> x + y);
```

Comment:
```suggestion
        return numbers.stream()
        .filter(sumStrategy::isSuitable)
        .reduce(0, (x, y) -> x + y);
```
스트림은 기능단위로 줄바꿈하여 사용하는것을 권장 드립니다 !